server/routerlicious/packages/lambdas/src/index.ts

```diff
@@ -3,7 +3,12 @@
  * Licensed under the MIT License.
  */
 
-export { configureWebSocketServices } from "./alfred";
+export {
+	configureWebSocketServices,
+	CollaborationSessionEventType,
+	ICollaborationSessionEvent,
+	ICollaborationSessionEvents
```

Comment:
Do we have a use case in mind for exporting the new bits from the package? Otherwise I think we shouldn't. Applies to `createRuntimeMessage` below too.

Reply:
Don't we need to export `createRuntimeMessage` from utils for alfred to use it though?

Reply:
Oh we can also consume `createRuntimeMessage` by reaching in to the file through "../utils". True.